### ISSUE TITLE
TP servers page:  quick filter should only apply to currently viewed colums.

### ISSUE THREAD

**Issue by traeak** [apache/trafficcontrol#5432]
## I'm submitting a ...

-  improvement request (usability, performance, tech debt, etc.)

## Traffic Control components affected ...

-  Traffic Portal

## Current behavior:

The "quick search" box filters on all columns, not just the columns currently selected for display.

## New behavior:

The "quick search" box should only apply to columns currently selected for viewing.

## Minimal reproduction of the problem with instructions:

During an upgrade I took a batch of machines to admin_down with reason that included a substring of the new profile.
With the "offline reason" column turned off (not being viewed), quick search for that new profile substring.
quick search results include not only servers on the new profile but also also all the servers with the "offline reason" with the new profile.

**Comment by mitchell852** [comment]
yeah, it can be confusing when you get matches on data that has been hidden. fyi - this applies to any TP table that uses the ag-grid: servers, ds requests, jobs, etc.
